src/server/db_slice.cc

```diff
@@ -1311,17 +1311,24 @@ auto DbSlice::DeleteExpiredStep(const Context& cntx, unsigned count) -> DeleteEx
   std::string stash;
 
   auto cb = [&](ExpireIterator it) {
-    auto key = it->first.GetSlice(&stash);
+    string_view key = it->first.GetSlice(&stash);
     if (!CheckLock(IntentLock::EXCLUSIVE, cntx.db_index, key))
       return;
 
     result.traversed++;
     time_t ttl = ExpireTime(it) - cntx.time_now_ms;
     if (ttl <= 0) {
       auto prime_it = db.prime.Find(it->first);
-      CHECK(!prime_it.is_done());
-      result.deleted_bytes += prime_it->first.MallocUsed() + prime_it->second.MallocUsed();
-      ExpireIfNeeded(cntx, prime_it);
+      if (prime_it.is_done()) {  // A workaround for the case our tables are inconsistent.
```

Comment:
Do we understand under what conditions the tables are inconsistent ?

Reply:
no